packages/aws-lambda/test/infra-mock/index.js

```diff
@@ -35,10 +35,10 @@ const legacySensorMode = process.env.LEGACY_SENSOR != null;
 const anotherLambda = process.env.ANOTHER != null;
 const onlyLatest = process.env.ONLY_LATEST;
 
-const name = process.env.LAMBDA_FUNCTION_NAME || 'wrapped-async';
+const name = process.env.LAMBDA_FUNCTION_NAME || 'nodejs-tracer-lambda';
```

Comment:
Renamed our Lambda function to 'nodejs-tracer-lambda' for easier identification as it's used in our integration tests. Additionally, I've added the necessary tags to the function.

Reply:
🤔 Could you please expand on "easier identification"?

Reply:
Since we're sharing the aws dev account with other teams, it’s easier to identify the Lambda function that is used by our team.

Reply:
Yes the prefix makes sense, but changing the function name ? 🤔